### PR TITLE
Custom Resources Article

### PR DESCRIPTION
**Pull request by Manchick0**
Introduced a comprehensive guide to custom resources, covering client and server-side data.

### CODE REVIEW

File: develop/custom-resources.md [hunk 1]
```diff
@@ -0,0 +1,126 @@
...
+
+> We'll create a `JsonDataLoader` for loading JSON files from the `fruit` folder
+> in a data pack, deserialize them into actual fruits, and store them
+> for further retrieval.
+
+Let's start by creating a class itself. It should extend `JsonDataLoader`, and override the constructor
+and the `apply` method.
+
+```java
+public class FruitDataLoader extends JsonDataLoader {
```
Comment (IMB11) [inline develop/custom-resources.md]: Needs to be in the reference mod. Just because it uses mc classes which may change in the future (ResourceManager)

**Review by IMB11 — CHANGES_REQUESTED** [review]
Really good, just a minor nitpick.

File: develop/custom-resources.md (hunk 1 above)
Comment (Manchick0) [reply develop/custom-resources.md]: Sure! I added a new class, which is just empty. I really didn't want to do it though, that's why I decided to use raw code in the previous version.

**Comment by Manchick0** [comment]
Any updates?

**Pull request closed without merging**